tests/store/test_store_export.py

```diff
@@ -160,13 +160,35 @@ def test_compare():
     assert compare_dict == {'rs1': False, 'rs2': True}
 
 
-def test_get_equality_wrong_key():
-    # GIVEN two analysis dicts like this:
-    analysis_1 = {'rs1': ['T', 'C'], 'rs2': ['A', 'G']}
-    analysis_2 = {'rs5': ['A', 'C'], 'rs2': ['A', 'G']}
+def test_get_analysis_equalities_one_analysi_missing(genotype_db):
+    # GIVEN a sample with two analyses
 
-    # WHEN running _get_equality
-    compare_dict = _get_equality(analysis_1, analysis_2)
+    sample_id = 'test'
```

Comment:
Would it be possible to have a populated genotype_db as a fixture? Maybe this would be a handy fixture in future tests, and the test code would look a bit cleaner.

Reply:
True. Ill look at it.